pkg/proxy/logger/endpoint/epinfo.go

```diff
@@ -59,23 +66,38 @@ func (r *endpointInfoRegistry) FillEndpointInfo(info *accesslog.EndpointInfo, ad
 	// Only resolve the security identity if not passed in, as it may have changed since
 	// reported by the proxy. This way we log the security identity and labels used for
 	// policy enforcement, if any.
-	if id == 0 {
+	if info.Identity == 0 {
+		// Try and look up identity by endpoint
 		if ep != nil {
-			id = ep.GetIdentity()
-		} else if addr.IsValid() {
+			info.ID = ep.GetID()
```

Comment:
This might be unnecessary. If `ep != nil` here, then `info.ID` should already be set by the above block right?

Reply:
oh derp, good point.